client/js/_author/components/assessments/question_types/drag_and_drop/_drag_and_drop.jsx

```diff
@@ -0,0 +1,96 @@
+import React              from 'react';
+import { connect }        from 'react-redux';
+import * as AssetActions  from '../../../../../actions/qbank/assets';
+import TargetArea         from './target_area';
+import DragArea           from './drag_area';
+import Feedback           from '../question_common/single_feedback';
+import localize           from '../../../../locales/localize';
+
+function select(state) {
+  return {
+    loadingMedia: state.media.loading,
+    images: state.media.image,
+  };
+}
+
+export class DragAndDrop extends React.Component {
+  static propTypes = {
+    item: React.PropTypes.shape({
+      id: React.PropTypes.string,
+      bankId: React.PropTypes.string,
+      question: React.PropTypes.shape({}),
+    }).isRequired,
+    images: React.PropTypes.shape({}).isRequired,
+    updateItem: React.PropTypes.func.isRequired,
+    addMediaToQuestion: React.PropTypes.func.isRequired,
+    updateChoice: React.PropTypes.func.isRequired,
+    // localizeStrings: React.PropTypes.func.isRequired,
+    loadingMedia: React.PropTypes.bool,
+  };
+
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  uploadMedia(file, where, metadata, newMedia) {
+    this.props.addMediaToQuestion(
+      file,
+      this.props.item.bankId,
```

Comment:
I already have to two line that function, I am not going to add more props to it